programs/locker/src/util/remaining_accounts.rs

```diff
@@ -56,23 +52,11 @@ pub fn parse_remaining_accounts<'a, 'info>(
         *remaining_accounts = &remaining_accounts[end_idx..];
 
         match slice.accounts_type {
-            AccountsType::TransferHookInput => {
-                if parsed_remaining_accounts.transfer_hook_input.is_some() {
+            AccountsType::TransferHookEscrow => {
```

Comment:
Why do we still need `AccountsType` when it supports only single token mint?

Reply:
I think let's keep it as it is so it's more generic, in case we want to support more features with additional remaining accounts.